src/modules/dpos/validator/api/index.js

```diff
@@ -37,40 +39,40 @@ const getDelegateProps = ({ address, publicKey, username }) => {
  * @param {Object} data.network - Network setting from Redux store
  * @returns {Promise} http call
  */
-export const getDelegate = ({
-  params = {}, network, baseUrl,
-}) => http({
-  path: httpPaths.delegates,
-  params: { ...getDelegateProps(params), isDelegate: true },
-  network,
-  baseUrl,
-});
+export const getDelegate = ({ params = {}, network, baseUrl }) =>
+  http({
+    path: httpPaths.delegates,
+    params: { ...getDelegateProps(params), isDelegate: true },
+    network,
+    baseUrl,
+  });
```

Comment:
In general, please refrain from reformatting the code which you do not refactor. This will cause unnecessary conflicts, increases the number of files you changed by your PR, making the review a longer task. We'll change it when we need to change it.